pom.xml

```diff
@@ -976,6 +976,7 @@ under the License.
     <version.sisu-maven-plugin>0.9.0.M4</version.sisu-maven-plugin>
     <version.plexus-utils>4.0.2</version.plexus-utils>
     <version.spotless-maven-plugin>3.0.0</version.spotless-maven-plugin>
```

Comment:
imho, we are using spotless @elharo 

- https://github.com/diffplug/spotless/pull/2651#discussion_r2384089886